KeyboardKit/KeyboardShortcuts.swift

```diff
@@ -0,0 +1,52 @@
+#if canImport(SwiftUI)
+import SwiftUI
+
+@available(iOS 14.0, *)
+public extension KeyboardShortcut {
+    
+    /// The standard keyboard shortcut for a save action, consisting of the 'S' key and the Command (⌘) modifier.
+    static let saveAction: Self = .init("s", modifiers: .command)
```

Comment:
Did you consider setting this up somehow to avoid repeating that cmd S is save both here and for `KeyboardBarButtonItem`? E.g. a basic (messy) way to do that would be to actually make a `KeyboardBarButtonItem` here and ask to it provide it’s key command, and use the properties of that key command to create the `SwiftUI.KeyboardShortcut`. This could probably be better with some refactoring in `KeyboardBarButtonItem` such as adding a KeyboardKit internal helper.

I think it would be nice to have the source of truth consolidated into one place.

Reply:
Ah yep, there's definitely some duplication that could be tidied up here. I'll see what I can work out